tfx/types/artifact_utils.py

```diff
@@ -98,7 +98,12 @@ def get_split_uris(artifact_list: List[Artifact], split: Text) -> List[Text]:
     split_names = decode_split_names(artifact.split_names)
     if split in split_names:
       matching_artifacts.append(artifact)
```

Comment:
We could put the os.path.join() call here and rename matching_artifacts to result (return it in the end)